packages/synapse-react-client/src/components/ColumnFilter/ColumnFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { useQueryContext } from '../QueryContext/QueryContext'
+import {
+  FacetColumnResultValues,
+  FacetColumnValuesRequest,
+} from '@sage-bionetworks/synapse-types'
+import { useQuery } from '@tanstack/react-query'
+import { Autocomplete, TextField } from '@mui/material'
+import { getCorrespondingSelectedFacet } from '../../utils/functions/queryUtils'
+
+export type FilterProps = {
+  filterColumnName: string
+}
```

Comment:
This slipped my mind before, but since this is a facet column, can we update this:

```suggestion
export type FilterProps = {
  topLevelEnumeratedFacetToFilter: UniqueFacetIdentifier
}
```

Not 100% sure about `topLevelEnumeratedFacetToFilter` but I want to make it clear that this must be an enumerated facet, and providing the value adds the 'top-level filter' to the UI